app/views/questions/show.html.erb

```diff
@@ -45,7 +45,7 @@
           <%= f.hidden_field :question, value: params[:id] %>
           <%= f.text_area :body, placeholder: "Add Comment" %>
           <%= show_errors(@comment, :body) %>
-          <%= f.submit "Add Comment" %>
+          <%= f.submit "Add Comment", class: "white-text waves-effect blue btn-small" %>
```

Comment:
I like the standardization you built in for making all of the buttons look spiffy. It adds a lot of continuity to the site while making it spiffier.